core/libs/aspect-exporter/src/lib/rdf-list/rdf-list.service.ts

```diff
@@ -80,6 +80,38 @@ export class RdfListService implements CreateEmptyRdfList, EmptyRdfList {
     return this;
   }
 
+  pushEntityValueLangString(
+    source: SourceElementType,
+    ...elements: Array<{
+      predicate: NamedNode;
+      literal: Quad_Object;
+    }>
+  ) {
+    const subject = DataFactory.namedNode(source.aspectModelUrn);
+    const preparedList = elements.map(({predicate}) => {
+      const {list, created} = this.getListOrCreateNew(subject, predicate);
+      return {list, created, predicate};
+    });
+
+    if (!preparedList.length) {
+      return this;
+    }
+
+    preparedList
+      .filter(list => list.created)
+      .forEach(({list, predicate}) => {
+        const listElements = this.getListElements(list);
+        const elementsToBeAdded = elements.filter(element => element.predicate.id === predicate.id).map(({literal}) => literal);
+
+        if (elementsToBeAdded.length) {
+          // this.remove(source, ...elements);
```

Comment:
Please remove this comment, when not used.

Reply:
done